docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        stdout = open(deseq_resdir + "/" + deseq_prefix + ".log", 'w')
</pre><p>minor comment: </p><p> </p><p>in general, using 'with` statements is preferred as you can't forget to close a file</p><pre class="ql-syntax" spellcheck="false">with open(deseq_resdir + "/" + deseq_prefix + ".log", 'w') as stdout:
    with open(deseq_resdir + "/" + deseq_prefix + ".err", 'w') as stderr:
        run(...)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>